assets/js/lib/lists/index.js

```diff
@@ -11,13 +11,15 @@ export const hasOne = (elements, list) =>
     false
   );
 
-export const pages = (list) => {
-  const hasRest = Boolean(list.length % 10);
-  return hasRest ? ~~(list.length / 10) + 1 : ~~(list.length / 10);
+export const pages = (list, itemsPerPage = 10) => {
+  const hasRest = Boolean(list.length % itemsPerPage);
+  return hasRest
+    ? ~~(list.length / itemsPerPage) + 1
+    : ~~(list.length / itemsPerPage);
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏

TIL: `~~` is `Math.floor`

Reply:
(I have to admit I just coded this after looking at some very dark corner on the internet)